Gems/Atom/RHI/DX12/Code/Source/RHI/CommandList.h

```diff
@@ -483,17 +503,6 @@ namespace AZ
                         {
                             GetCommandList()->SetComputeRootDescriptorTable(binding.m_samplerTable.GetIndex(), compiledData.m_gpuSamplersDescriptorHandle);
                         }
```

Comment:
Hmm. The old way of direct binding unbounded arrays is gone? I guess then we cant check this code in untill Terrain and RT switches over correct?

Reply:
Yea, unless we make additional changes to this PR to support it. It was difficult for me to test both flows at once since the RT shaders don't have these changes and the exe crashes if they are enabled